Crawler/Analyzers/UtilAnalyzers/PunctuationAnalyzer.cs

```diff
@@ -80,5 +91,45 @@ private List<int> WordsCountBetweenPunctuations(List<Token> tokens)
 
             return wordsCount;
         }
+
+        public double CalculatePassiveVoiceSentencesPercentage(List<PosTagToken> posTagTokens)
+        {
+	        var toBeForms = ExtractWordsFromFile(config.Value.ToBeFormsFile);
+	        var passiveVoiceSentencesCount = 0;
+	        var indexOfNextSentenceEnd = posTagTokens.FindIndex(token => token.ExtendedType == ".");
+            var totalSentencesCount = posTagTokens.Count(token => token.ExtendedType == ".");
```

Comment:
need to shift right

Reply:
I don't know why it's looks like this in here but in VS it's fine:
![image](https://user-images.githubusercontent.com/21021560/74934119-2073c000-53ee-11ea-930e-146b44a852c1.png)